.build/BuildHelper/Build-FinOpsModule.ps1

```diff
@@ -0,0 +1,82 @@
+function Build-FinOpsModule
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [string]
+        [ValidateScript({$_ -match '^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'})]
+        $Version,
+
+        [Parameter()]
+        [ValidateSet('alpha', 'preview')]
+        [string]
+        $PrereleaseTag
+    )
+
+    $rootPath = (Get-Item -Path $PSScriptRoot).Parent.FullName
+    $moduleName = 'FinOpsToolkit'
+    $moduleFullName = "$moduleName.psm1"
+    $modulePath = Join-Path -Path $rootPath -ChildPath "src/powershell/$moduleFullName"
+    $stringsPath = Join-Path -Path $rootPath -ChildPath 'src/powershell/en-US'
+    $releasePath = Join-Path -Path $rootPath -ChildPath "release/FinOpsToolkit/$Version"
```

Comment:
Should we label this as PowerShell? Is there ever a scenario where we'd build an old version that isn't the current one?

Reply:
Version will be computed at build time with [GitVersion](https://gitversion.net/url). At least that's what I was planning on doing.

Reply:
My point is that we'll only ever build one version and everything in the release folder is that same version, so it probably isn't needed to add the version as a subfolder. Just a nit, I suppose.

Reply:
Oh I see what you mean. Modules have versioned subfolders. This way you can have multiple versions of any one module installed together